identify-layer-features/src/main/java/com/esri/arcgismaps/sample/identifylayerfeatures/screens/MainScreen.kt

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.identifylayerfeatures.screens
+
+import android.app.Application
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.esri.arcgismaps.sample.identifylayerfeatures.components.ComposeMapView
+import com.esri.arcgismaps.sample.identifylayerfeatures.components.MapViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String, application: Application) {
+    // coroutineScope that will be cancelled when this call leaves the composition
+    val sampleCoroutineScope = rememberCoroutineScope()
+    // create a ViewModel to handle MapView interactions
+    val mapViewModel = remember { MapViewModel(application, sampleCoroutineScope) }
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it)
+            ) {
+                // composable function that wraps the MapView
+                ComposeMapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    mapViewModel = mapViewModel
+                )
+                // Bottom text to display the identify results
+                Row(
+                    modifier = Modifier
+                        .padding(12.dp)
+                        .fillMaxWidth()
+                ) {
```

Comment:
The bottom row is causing a resize to both the `Row` and `ComposeMapView`, this looks janky without any transitions. A quick fix would be to set the Modifier to use `animateContentSize()`, which should help resize both elements with transitions. 

```suggestion
                ComposeMapView(
                    modifier = Modifier
                        .fillMaxSize()
                        .weight(1f)
                        .animateContentSize(),
                    mapViewModel = mapViewModel
                )
                // Bottom text to display the identify results
                Row(
                    modifier = Modifier
                        .padding(12.dp)
                        .fillMaxWidth()
                        .animateContentSize()
                ) {
```